README.adoc

```diff
@@ -815,14 +828,10 @@ deployments, systems, and routes:
 oc delete -f deploy.yaml
 ```
 
-To completely delete your Minishift VM, cluster, and all associated files run the following command:
-
-[role=command]
-```
-minishift delete
-```
+To completely delete your Minishift VM, cluster, and all associated files, refer to the official 
+https://docs.okd.io/latest/minishift/getting-started/uninstalling.html[Uninstalling Minishift^] documentation.
 
-To revert your Docker daemon back to its default settings, simply reboot your system.
+To disconnect from Minishift's internal Docker daemon and reconnect to your local daemon, simply close your shell session.
```

Comment:
There isn't actual disconnection and reconnection, right?  Closing the shell will get rid of the environment settings and starting a new one will give you your default ones?